packages/familie-tidslinje/src/components/tidslinje/TimelinePeriod.tsx

```diff
@@ -92,6 +261,22 @@ const finnClassnames = (
     const classNames = [];
 
     switch (period.cropped) {
+        case 'both':
+            classNames.push('croppedBegge');
+            break;
+        case 'left':
+            if (period.direction === 'left') classNames.push('croppedVenstre');
+            else classNames.push('croppedHøyre');
+            break;
+        case 'right':
+            if (period.direction === 'left') classNames.push('croppedHøyre');
+            else classNames.push('croppedVenstre');
+            break;
```

Comment:
Jeg tror ikke jeg helt skjønner dette 😅  Hvis `period.cropped = 'left'` får vi klassen `croppedHøyre` hvis `period.direction != 'left'`?

Reply:
Går på om tidslinjen skal vise perioden stigende eller synkende. Så stigende => direction == 'left'.

Reply:
ser også at direction blir brukt for å plassere diverse ting i css

Reply:
Ok, det var bare forvirrende å lese 🤔